README.md

```diff
@@ -12,7 +12,8 @@ Procurement is a Windows application for the online game Path Of Exile, that log
 * Search for gear based on item stats
 * Show where items are in your stash, that form part of vendor recipes
 * View your characters/gear
-* Oneclick generate forum trade posts that you can copy&paste into a trade post, which will have all your gear linked, verified and categorised in spoiler tags.
+* Oneclick generate forum trade posts that you can copy&paste into a trade post, which will have all your gear linked, verified and categorised in spoiler tags
+* Generate ![Path of Building](https://github.com/openarl/pathofbuilding) compatible item definitions
```

Comment:
Should have a hyphen at least between "Building" and "compatible".

Reply:
I'm not sure what you mean exactly. This looks ok to me.
![image](https://user-images.githubusercontent.com/9460838/52699008-aacc4d00-2f6c-11e9-8016-8175ac37f7cc.png)

Reply:
The phrase "Path of Building compatible" is a [compound adjective](https://www.quickanddirtytips.com/education/grammar/how-to-use-a-hyphen).  I was suggesting to change it to "Path of Building-compatible".  According to [some style guides](https://www.btb.termiumplus.gc.ca/tcdnstyl-chap?lang=eng&lettr=chapsect2&info0=2.04), though, when the compound adjective is made up of a noun and an adjective, the hyphen is only necessary when the phrase is otherwise ambiguous, which it isn't here.